drone/exec.go

```diff
@@ -169,9 +171,14 @@ func execCmd(c *cli.Context) error {
 			if err != nil {
 				return err
 			}
-			payload.Workspace.Keys = &drone.Key{
+			payload.Keys = drone.Key{
 				Private: string(key),
 			}
+			payload.Netrc = drone.Netrc{
+				Machine: "",
+				Login: "",
+				Password: "",
+			}
```

Comment:
@greenboxal you actually don't need to explicitly set the values to empty strings. This is because strings default to their zero value, which is an empty string. So you can actually make the following change and it will behave the same way:

```diff
-payload.Netrc = drone.Netrc{
-	Machine: "",
-	Login: "",
-	Password: "",
-}
+payload.Netrc = drone.Netrc{}
```